internal/services/sentinel/sentinel_data_connector_microsoft_threat_intelligence.go

```diff
@@ -57,6 +58,7 @@ func (s DataConnectorMicrosoftThreatIntelligenceResource) Arguments() map[string
 		},
 
 		"bing_safety_phishing_url_lookback_date": {
+			Deprecated:   "This field is deprecated and will be removed in version 4.0 of the AzureRM Provider.",
```

Comment:
Is the block here and [here](https://github.com/hashicorp/terraform-provider-azurerm/pull/21954/files#diff-e55bd9e1d873da0245e2ff8bc94d56aa97e053a58387ba539dd75b8e0bc81c0eR78) the same? if so, we can remove this block in favor of the block down below.